cli/popper/runner_host.py

```diff
@@ -136,7 +136,7 @@ def run(self, step):
         container.start()
         cout = container.logs(stream=True)
         for line in cout:
-            log.step_info(pu.decode(line).strip('\n'))
+            log.step_info(pu.decode(line).rstrip())
```

Comment:
the utility function was introduced to deal with compatibilities between python 2 and 3. We don't support python 2 now, so we can remove that and invoke `decode` directly on the `bytes` variable